css/about.css

```diff
@@ -111,14 +111,14 @@
   opacity: 0.9;
   text-align: center;
   text-decoration: underline solid #272a31;
-  margin-bottom: 20px;
+  padding-bottom: 20px;
 }
 
 /* starting main section for mobile screen */
 .main {
   background-color: #fff;
   width: 100%;
-  margin-bottom: 100px;
+  margin-bottom: 20px;
 }
```

Comment:
- [ ] Kindly make sure for your whole app you respect the fonts provided by Cindy, especially for the titltes :recycle: 

![Screenshot from 2023-02-09 10-19-57](https://user-images.githubusercontent.com/97834160/217758391-11a729e5-ed87-4344-a070-b6fdfd726eb9.png)


![Screenshot from 2023-02-09 10-17-43](https://user-images.githubusercontent.com/97834160/217758496-650e1acf-5316-463b-9a9f-c2ce73d38e6e.png)